src/distr/other.rs

```diff
@@ -118,6 +118,7 @@ impl Distribution<char> for StandardUniform {
         if n <= 0xDFFF {
             n -= GAP_SIZE;
         }
+        // SAFETY: We ensure above that `n` represents a `char`.
```

Comment:
Maybe add a `debug_assert!` here that uses `char`'s impl of `TryFrom<u32>` to confirm?

Reply:
Digging through libstd code (`core/char/convert.rs`), it looks like it already does this. In any case, it's not particularly hard to prove this case using local reasoning.